src/Microsoft.IdentityModel.Tokens/Base64UrlEncoding.cs

```diff
@@ -388,6 +406,68 @@ private static int ValidateAndGetOutputSize(string inputString, int offset, int
             return outputsize;
         }
 
+        /// <summary>
+        /// Validates the input span for decode operation.
+        /// </summary>
+        /// <param name="strSpan">String represented by a span to validate.</param>
+        /// <param name="offset">Index of char in <paramref name="strSpan"/> to start decode operation.</param>
+        /// <param name="length">Number of chars in <paramref name="strSpan"/> to decode, starting from offset.</param>
+        /// <returns>Size of the decoded bytes arrays.</returns>
+        internal static int ValidateAndGetOutputSize(ReadOnlySpan<char> strSpan, int offset, int length)
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

this should be covered by it's own unit tests

Reply:
run fine code coverage on one of readToken tests...I think it would be best to just unit test this to the max, and not test through other tests.

Reply:
I will add unit tests once we align on the approach.

Reply:
Added tests for this method